op-batcher/batcher/channel_manager.go

```diff
@@ -64,6 +66,7 @@ func (s *channelManager) Clear() {
 	defer s.mu.Unlock()
 	s.log.Trace("clearing channel manager state")
 	s.blocks = s.blocks[:0]
+	s.l1OriginLastClosedChannel = nil
```

Comment:
Could pass it as a parameter to `Clear` so it's always set. Would then also set it at startup.